int16 taskId
int16 priority
int16 taskType
int16 payload
int16 taskState
float32 cost
float32 energyCost
int16 robotId
geometry_msgs/Pose[] pose